doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -68,20 +68,22 @@ different. This can be seen when the :flag:`Printing Universes` flag is on:
    Print selfpid.
 
 Now :g:`pidentity` is used at two different levels: at the head of the
-application it is instantiated at ``Top.3`` while in the argument position
-it is instantiated at ``Top.4``. This definition is only valid as long as
-``Top.4`` is strictly smaller than ``Top.3``, as shown by the constraints. Note
-that this definition is monomorphic (not universe polymorphic), so the
-two universes (in this case ``Top.3`` and ``Top.4``) are actually global
-levels.
+application it is instantiated at ``u`` while in the argument position
+it is instantiated at ``u0``. This definition is only valid as long as
+``u0`` is strictly smaller than ``u``, as shown by the constraints.
+Note that if we made ``selfpid`` universe monomorphic, the two
+universes (in this case ``u`` and ``u0``) would be declared in the
+global universe graph with names ``selfpid.u0`` and ``selfpid.u1``.
+Since the constraints would also be global ``Print selfpid.`` would
+not show them and we would have to use :cmd:`Print Universes`.
```

Comment:
```suggestion
Since the constraints would be global, ``Print selfpid.`` will
not show them, however they will be shown by :cmd:`Print Universes`.
```